functional/streams.py

```diff
@@ -174,8 +175,28 @@ def json(json_file):
         return seq(six.viewitems(json_input))
 
 
+def sqlite3(conn, sql):
```

Comment:
Use additional optional positional/keyword arguments to pass options to `sqlite3api.connect`, or `conn.execute`